app/build.gradle

```diff
@@ -16,6 +16,9 @@ android {
         release {
             minifyEnabled true
         }
+        debug {
```

Comment:
Won't this make debug builds impossible to debug?

Reply:
Thats right! We had to enable this to address the obfuscation issue. The test app needed to behave close to the production app to detect 422 error because of obfuscated events and we do not build release version for test app.
Locally we have been disabling minify and building for debugging. This has been a bit painful and @nkukday brought this up. 
We would address this by reverting this and adding another debugMinify{minifyEnabled true} to test obfuscated events.